control-plane/subcommand/connect-init/command.go

```diff
@@ -70,6 +76,8 @@ func (c *Command) init() {
 	c.flagSet.StringVar(&c.flagACLTokenSink, "acl-token-sink", defaultTokenSinkFile, "File name where where ACL token should be saved.")
 	c.flagSet.StringVar(&c.flagProxyIDFile, "proxy-id-file", defaultProxyIDFile, "File name where proxy's Consul service ID should be saved.")
 	c.flagSet.BoolVar(&c.flagMultiPort, "multiport", false, "If the pod is a multi port pod.")
+	c.flagSet.BoolVar(&c.flagGateway, "gateway", false, "If the pod is a Consul gateway pod.")
+	c.flagSet.StringVar(&c.flagGatewayKind, "gateway-kind", "", "Name of the gateway that is being registered.")
```

Comment:
```suggestion
	c.flagSet.StringVar(&c.flagGatewayKind, "gateway-kind", "", "Kind of the gateway that is being registered.")
```

Reply:
I actually fixed this in my PR.

Reply:
Nice! Feel free to ignore this one then @thisisnotashwin

Reply:
i have changed this. sorry.